src/ts/components/core/Switch.tsx

```diff
@@ -41,6 +41,8 @@ interface Props
     disabled?: boolean;
     /** State of check box */
     checked?: boolean;
+    /** dashRenderType */
+    dashRenderType?: any;
```

Comment:
Where does this come from?  If it's an internal thing, does it have to be a prop?  If it needs to be a prop, can it have a type other than `any`?

Reply:
Can you update the comment to include something like "For internal use only. Do not set or override this prop manually."  and change the type to string?

Reply:
What do you mean?